Documentation/op-guide/maintenance.md

```diff
@@ -62,7 +62,7 @@ The space quota can be triggered with a loop:
 
 ```sh
 # fill keyspace
-$ while [ 1 ]; do dd if=/dev/urandom bs=1024 count=1024  | etcdctl put key  || break; done
+$ while [ 1 ]; do dd if=/dev/urandom bs=1024 count=2  | etcdctl put key  || break; done
```

Comment:
The point is more to trigger the alarm than to fill it completely. There can still be some space left even with 2k puts.

`etcdctl set` won't work; the quotas only apply to the v3 backend. Probably `ETCDCTL_API=3 etcdctl put key` instead?